packages/lshw.sh

```diff
@@ -0,0 +1,50 @@
+LSHW_VERSION="B.02.20"
+LSHW_CHECKSUM="5805eba5f31886582fff673c5dccdb3b"
+LSHW_LINK="https://github.com/lyonel/lshw/archive/refs/tags/${LSHW_VERSION}.tar.gz"
+
+download_lshw() {
+    download_file $LSHW_LINK $LSHW_CHECKSUM
+}
+
+extract_lshw() {
+    if [ ! -d "lshw-${LSHW_VERSION}" ]; then
+        echo "[+] extracting: lshw-${LSHW_VERSION}"
+        tar -xf ${DISTFILES}/${LSHW_VERSION}.tar.gz -C .
+    fi
+}
+
+prepare_lshw() {
+    echo "[+] configuring lshw"
+    export CC="${CC:-gcc}"
+    export CXX="${CXX:-g++}"
+    export CFLAGS="${CFLAGS:--Os -fomit-frame-pointer}"
+    export CXXFLAGS="${CXXFLAGS:--Os -fomit-frame-pointer}"
+}
+
+compile_lshw() {
+    pushd src
+    make
```

Comment:
You should keep `${MAKEOPTS}`, it's on that variable that amount of build jobs is set and you want to keep it parallelized :)